src/util/util.js

```diff
@@ -15,47 +1,61 @@
+module.exports = class Util {
+  static color (color) {
+    if (typeof color === 'string') {
+      if (color.toLowerCase() === 'random') return Math.floor(Math.random() * (0xffffff + 1))
+      if (color.toLowerCase() === 'default') return 0
+      color = parseInt(color.replace('#', ''), 16)
+    } else if (Array.isArray(color)) {
+      color = (color[0] << 16) + (color[1] << 8) + color[2]
     };
 
-  static parseEmoji(text = '') {
-    if (text.includes('%')) text = decodeURIComponent(text);
-    if (!text.includes(':')) return { animated: false, name: text, id: null };
-    const m = text.match(/<?(?:(a):)?(\w{2,32}):(\d{17,19})?>?/);
-    if (!m) return null;
-    return { animated: Boolean(m[1]), name: m[2], id: m[3] || null };
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Shibe-Studio/Shynk/pullRequest?prid=7542588)
```suggestion
    if (color < 0 || color > 0xffffff) throw new Error("A cor deve ser um código hex!")
```